main/main.cpp

```diff
@@ -3461,8 +3420,6 @@ Error Main::setup2(bool p_show_boot_logo) {
 
 	MAIN_PRINT("Main: Load Navigation");
 
```

Comment:
```suggestion

		NavigationServer3DManager::initialize_server(); // 3D server first because 2D depends on it.
		NavigationServer2DManager::initialize_server();

```